tox.ini

```diff
@@ -53,13 +52,12 @@ extras =
     alldeps: all
 
 commands =
-    # FIXME: Not sure why need this for tox to see the C-ext
-    python setup.py build_ext --inplace
-    # End of FIXME
     pip freeze
     !cov: pytest --pyargs synphot {toxinidir}/docs {posargs}
-    cov: pytest --pyargs synphot {toxinidir}/docs --cov synphot --cov-config={toxinidir}/setup.cfg {posargs}
-    cov: coverage xml -o {toxinidir}/coverage.xml
+    # FIXME: Seems like using -c option with multiline Pytest is broken: see
+    # https://github.com/tox-dev/tox-travis/issues/146
+    # at the very least on CI - it runs fine locally on MacOSX.
+    cov: pytest --rootdir={toxinidir} -c {toxinidir}/setup.cfg --pyargs synphot --cov={envsitepackagesdir}/synphot --cov-config={toxinidir}/setup.cfg {posargs} --cov-report xml:{toxinidir}/coverage.xml
```

Comment:
Can you explain why the changes here?

Reply:
Not really. It seems like the directory in which `pytest` is going to run depends on a combination of factors such as `rootdir`, location of the config for pytest, etc. I was just lucky enough to guess the right incantation.